parm/config/config.prepbufr

```diff
@@ -11,9 +11,9 @@ echo "BEGIN: config.prepbufr"
 # Set variables
 
 if [ $machine = "HERA" ]; then
-    export GESROOT=/scratch1/NCEPDEV/rstprod   # set by module prod_envir on WCOSS_C
+    export GESROOT=/scratch1/NCEPDEV/rstprod
 elif [ $machine = "ORION" ]; then
-   export GESROOT=/dev/null
+    export GESROOT=/dev/null
```

Comment:
@KateFriedman-NOAA Didn't we get rstprod on Orion? Is there a GESROOT path to use now?

Reply:
While we do have rstprod on Orion that path isn't about rstprod stuff. The `GESROOT` on Hera is where NCO puts their 2-day mirror com outputs:
```
if [ $machine = "HERA" ]; then
    export GESROOT=/scratch1/NCEPDEV/rstprod   # set by module prod_envir on WCOSS_C
```

There isn't such a thing on Orion. I remember having to add `GESROOT` on Orion to avoid a path not getting set (in prep job I think), although it never gets used. It may no longer be needed and if so, can get removed from that config.

Reply:
Can remove in a later PR if it is deemed not used.